syft/frameworks/torch/tensors/interpreters/gradients.py

```diff
@@ -213,3 +213,13 @@ def gradient(self, grad):
         zero = self.self_ * 0
         gt_zero = self.self_ > zero
         return (gt_zero * grad,)
+
+class viewBackward(GradFunc):
+    def __init__(self, self_):
+        super().__init__(self,self_)
+        self.self_ = self_
+
+    def gradient(self, grad):
```

Comment:
Not sure, but isn't backward pass for `view` should just do `grad.view(old_shape)`?

Reply:
Yes. Can you check why test cases are failing ?